PowerPointLabs/PowerPointLabs/PositionsLab/PositionsLabMain.cs

```diff
@@ -361,7 +367,9 @@ public static void AlignCenter(ShapeRange toAlign, float slideHeight, float slid
                 case PositionsLabSettings.AlignReferenceObject.SelectedShape:
                     if (toAlign.Count < 2)
                     {
-                        throw new Exception(PositionsLabText.ErrorFewerThanTwoSelection);
+                        // Insert default align center behaviour here
+                        toAlign.Align(MsoAlignCmd.msoAlignMiddles, MsoTriState.msoTrue);
+                        toAlign.Align(MsoAlignCmd.msoAlignCenters, MsoTriState.msoTrue);
```

Comment:
Question: why the need to align twice?

Reply:
In order to align center, you need to first align horizontally and then vertically